tests/php/ORM/DataQueryTest.php

```diff
@@ -99,6 +101,33 @@ public function testApplyRelation()
         $this->assertStringContainsString('"testctwo_DataQueryTest_C"."ID" = "DataQueryTest_B"."TestCTwoID"', $dq->sql());
     }
 
+    public function provideApplyRelationMultiReciprocal()
```

Comment:
```suggestion
    public function provideApplyRelationMultiReciprocal(): array
```
Move provide method below test method

Reply:
Done